lib/test/tasks/run_file_size_checks.rb

```diff
@@ -17,7 +17,7 @@ class Test::Tasks::RunFileSizeChecks < Pallets::Task
     'groceries*.css' => (108..118),
     'groceries*.js' => (441..451),
     'home*.css' => (0..10),
-    'home*.js' => (260..270),
+    'home*.js' => (252..262),
```

Comment:
Reducing the page weight of the home app was a primary motivation for this change, so it's nice to see it paying off here (even if only in the form of ~8 KiB saved -- every bit counts).